### PR TITLE
fix

### PR DESCRIPTION
**Pull request by strengejacke**
# Description

This PR aims at adding this feature...

# Proposed Changes

I changed the `foo` function so that ...

### CODE REVIEW

File: R/report_sample.R [hunk 1]
```diff
@@ -132,6 +132,9 @@ report_sample <- function(data,
     insight::format_error("Cannot apply `weights` when grouping is done by more than one variable.")
   }
 
+  # make clean data frame
+  clas(data) <- "data.frame"
```
Comment (rempsyc) [inline R/report_sample.R:136]: `class` (typo)

**Comment by rempsyc** [comment]
Seems like mostly snapshot updates except those:

```
 ── Error ('test-report_sample.R:288:3'): report_sample, print vertical ─────────
  Error in `x[[jj]] <- v`: replacement has length zero
  Backtrace:
      ▆
   1. └─report::report_sample(iris_grp, select = 1:3) at test-report_sample.R:288:2
   2.   ├─base::`[<-`(`*tmp*`, , value = `<list>`) at report/R/report_sample.R:139:2
   3.   └─base::`[<-.data.frame`(`*tmp*`, , value = `<list>`) at report/R/report_sample.R:139:2
```

```
── Warning ('test-report.lm.R:38:3'): report.lm - lm intercept-only ────────────
Using `$` in model formulas can produce unexpected results. Specify your
  model using the `data` argument instead.
```

```
── Error ('test-report.lm.R:38:3'): report.lm - lm intercept-only ──────────────
Error in `eval(predvars, data, env)`: object 'd_wide' not found
```

```
── Error ('test-report.lm.R:38:3'): report.lm - lm intercept-only ──────────────
Error: Error: Unable to refit the model with standardized data.
```

**Comment by strengejacke** [comment]
Hm, the snapshots look wrong, though... I'll take a look later.

**Comment by rempsyc** [comment]
> [#375](https://github.com/easystats/report/pull/375) still has issues with snapshot tests. Tests via CI expect "2.93", but locally, it's "2.94", thus there's a mismatch between local and GitHub snapshots

If I remember correctly, that's why that test file was skipped at the top, and I think you've commented the skip. That makes sense, and it probably could be resolved by adding some tolerance level.

**Comment by rempsyc** [comment]
Thanks @strengejacke ! 😄

**Pull request merged**